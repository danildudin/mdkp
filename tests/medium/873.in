29 6 0
732 24 22 429 388 884 717 265 511 553 651 713 899 394 869 403 659 758 185 749 30 837 356 19 224 529 896 929 761 
65 464 153 213 858 383 200 547 98 229 588 783 246 349 129 989 353 622 624 584 760 205 604 533 10 956 767 653 953 
197 27 933 330 766 687 204 825 134 414 813 461 487 333 982 504 444 986 770 74 728 987 316 217 308 86 121 809 535 
938 965 131 617 498 508 767 40 243 440 555 608 611 56 455 664 625 266 252 451 229 587 293 668 175 448 663 918 573 
416 536 444 276 329 2 302 92 280 770 4 821 656 647 486 839 692 281 816 937 768 816 500 420 903 203 35 641 698 
505 40 606 503 527 390 401 65 16 745 409 756 982 110 839 486 391 491 406 128 441 829 254 619 936 329 886 106 860 
817 77 597 254 703 168 134 743 781 855 63 501 823 802 161 370 563 859 238 759 726 695 366 793 972 81 227 230 336 
8664 6181 5086 5503 8696 5536 
