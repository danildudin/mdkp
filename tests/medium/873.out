8086
12
5 8 10 12 13 14 17 18 19 25 26 28 
